lib/algo/NonuniformDMD.h

```diff
@@ -0,0 +1,78 @@
+/******************************************************************************
+ *
+ * Copyright (c) 2013-2022, Lawrence Livermore National Security, LLC
+ * and other libROM project developers. See the top-level COPYRIGHT
+ * file for details.
+ *
+ * SPDX-License-Identifier: (Apache-2.0 OR MIT)
+ *
+ *****************************************************************************/
+
+// Description: Computes the DMD algorithm on the given snapshot matrix. The
+//              implemented dynamic mode decomposition algorithm is derived from
+//              Tu et. al's paper "On Dynamic Mode Decomposition: Theory and
+//              Applications": https://arxiv.org/abs/1312.0041
+//              This algorithm also works in the case that the first sample does
+//              not start from t = 0.0 by incorporating a time offset.
```

Comment:
Could you update this description here?

Reply:
Thanks Kevin. The description of non-uniform DMD is added.